.gitignore

```diff
@@ -1,3 +1,4 @@
 vendor/*
 composer.lock
-.idea/*
\ No newline at end of file
+.idea/*
```

Comment:
`.idea` should be in a global gitignore. 

This should be removed in another PR.